src/logger.rs

```diff
@@ -112,33 +113,47 @@ pub fn get_derivability_results<L: SynthLanguage>(
     ruleset: &Ruleset<L>,
     derive_type: DeriveType,
     baseline: &Ruleset<L>,
+    derive: bool,
 ) -> ((usize, usize), (Duration, Duration), Value) {
-    let limits = if let DeriveType::AllRules = derive_type {
-        Limits {
-            iter: 2,
-            node: 100_000,
-        }
+    if !derive {
+        (
+            (0, 0),
+            (Duration::default(), Duration::default()),
+            json!({
+                "enumo_derives_baseline_derivable": vec![""],
+                "enumo_derives_baseline_underivable": vec![""],
+                "baseline_derives_enumo_derivable": vec![""],
+                "baseline_derives_enumo_underivable": vec![""],
+            }),
+        )
     } else {
-        Limits::deriving()
-    };
-
-    let start_f = Instant::now();
-    let (can_f, cannot_f) = ruleset.derive(derive_type, baseline, limits);
-    let time_f = start_f.elapsed();
-    let start_b = Instant::now();
-    let (can_b, cannot_b) = baseline.derive(derive_type, ruleset, limits);
-    let time_b = start_b.elapsed();
-
-    let derivability_results = json!({
-        "enumo_derives_baseline_derivable": &can_f.to_str_vec(),
-        "enumo_derives_baseline_underivable": &cannot_f.to_str_vec(),
-        "baseline_derives_enumo_derivable": &can_b.to_str_vec(),
-        "baseline_derives_enumo_underivable": &cannot_b.to_str_vec(),
-    });
-
-    (
-        (can_f.len(), can_b.len()),
-        (time_f, time_b),
-        derivability_results,
-    )
+        let limits = if let DeriveType::AllRules = derive_type {
+            Limits {
+                iter: 2,
```

Comment:
since we're not doing derive for halide on nightly anymore, can we put this back at 5?
or are we getting rid of All entirely?

Reply:
I think we're getting rid of it entirely.